src/helpers/userHelper.js

```diff
@@ -28,6 +28,9 @@ const timeOffRequest = require('../models/timeOffRequest');
 const notificationService = require('../services/notificationService');
 const { NEW_USER_BLUE_SQUARE_NOTIFICATION_MESSAGE } = require('../constants/message');
 const timeUtils = require('../utilities/timeUtils');
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+const cheerio = require('cheerio');
 
 const userHelper = function () {
   // Update format to "MMM-DD-YY" from "YYYY-MMM-DD" (Confirmed with Jae)
```

Comment:
Remove comment "(Confirmed with Jae)"

Reply:
That comment has not been created by me, so I don't suppose its a good idea to make changes to anything than the purpose of this PR.